scripts/testnet/prerequisites.sh

```diff
@@ -8,15 +8,15 @@ export DISTRIBUTION=$(cat /etc/os-release | grep "^ID=" | sed 's/ID=//')
 
 if [[ "$DISTRIBUTION" =~ ^(fedora|centos|rhel)$ ]]; then
   export PACKAGE_MANAGER="dnf"
-  export REQUIRED_PACKAGES="git golang gcc lsof jq curl"
+  export REQUIRED_PACKAGES="git golang gcc lsof jq curl tmux"
```

Comment:
Why did you change these?
You are affecting all other scripts/user scripts of normal chain with this.

How safe are these changes?
How compatible is it for other users using it?
Why are these required?

Reply:
1/ `tmux` is what we use, right?
2/ from what I know, we need golang 1.20 to be able to run the nodes, so we don't want to install the latest version
3/ clone that repo with SSH to be conistent

Reply:
tmux is optional actually, not required, we should be able to run nodes without it, not enforce it;
I think the best way to do it is to add that extra options with `tmux` only if the flag to `USETMUX` is enabled